test/unit/server/routes/health.spec.js

```diff
@@ -22,23 +28,33 @@ describe('GET /health', () => {
       }),
   );
 
-  context('when the build-info.json file is present', () => {
-    const projectRoot = path.resolve(__dirname, '../../../../');
-    const buildJson = path.resolve(projectRoot, 'build-info.json');
+  context('when the DB isn\'t working', () => {
```

Comment:
Oh, you mean not stubbing at all and just using the real thing? I do generally lean towards that but interacting with the filesystem directly felt a bit ugly for this sort of test.

There's still the REST test in the e2e pack for the health endpoint, I'll expand that to assert on some of the other bits